project/Settings.scala

```diff
@@ -111,9 +111,9 @@ object Settings {
 
     val apacheXMLRPC      = "3.1.3"
     val opencsv           = "2.1"
-    val epicsService      = "1.0.0"
-    val gmpCommandRecords = "0.7.0"
-    val guava             = "24.1-jre"
+    val epicsService      = "0.19"
+    val gmpCommandRecords = "0.6.6"
+    val guava             = "16.0.1"
```

Comment:
If they use the latest versions, how is it that the numbers go down ?

Reply:
a more detailed explanation is worthwhile. At the moment we have 2 branches for GMP. the 14.x line based on java 1.6 and the 15.x line base on java 1.8. We use the 15.x line on master at the moment towards supporting GIAPi. However I did my tests with the updated epics library on the 14.x line. Thus these version numbers are newer for the 14.x line.

Now that I think about it, it may be worthwhile to make the same change on 15.x and test, at least we'd be sure this is the only change, WDYT?

Reply:
I am a bit confused. Are we pulling epicsService from GMP 14.x ?